src/github.com/matrix-org/go-neb/services/rssbot/rssbot.go

```diff
@@ -349,17 +422,29 @@ func (s *Service) sendToRooms(cli *gomatrix.Client, feedURL string, feed *gofeed
 }
 
 func itemToHTML(feed *gofeed.Feed, item gofeed.Item) gomatrix.HTMLMessage {
+	body := fmt.Sprintf("<strong>%s</strong>:<br><a href=\"%s\"><strong>%s</strong></a>",
+		html.EscapeString(feed.Title), html.EscapeString(item.Link), html.EscapeString(item.Title))
```

Comment:
Please incorporate https://github.com/matrix-org/go-neb/pull/285/files

Reply:
Should https://github.com/matrix-org/go-neb/pull/311/files/97db7ae6dc21d03c11c430bd4aca2da5d233d016#diff-ae310978bc4c449759bda6db50889e71R440 also be using `itemTitle` instead of `item.Title`?

Reply:
Yes please.

Reply:
done!